anomalib/models/draem/perlin.py

```diff
@@ -0,0 +1,134 @@
+"""Helper functions for generating Perlin noise."""
```

Comment:
Can we move augmenter.py, perlin.py and loss.py to `components/` folder? This way we can keep the root directory clean.

Reply:
How about a `utils` directory? It would be more consistent with how we've structured the PatchCore directory. We could then use this convention for other models as well.